04-graphs/01-graphs.md

```diff
@@ -12,6 +12,26 @@ By the end of this lesson we should be able to:
   - Breadth First Search
   - Depth First Search
 
+### !callout-star
+
+## Practical Applications!
+
+Some common practical applications of graphs can be found below. As you learn more, try to imagine how graphs could be used to implement the following:
+
+- Mapping Services (Google Maps): Building transportation systems.
+- Social Media: Graphing connections between users.
+- Operating Systems: Use graphs to allocate resources.
+- World wide web: Graphing connections between web pages.
```

Comment:
Nitpick - for consistency, capitalize "Wide Web"

Reply:
Done!